cmake/CMakeLists.txt

```diff
@@ -937,3 +937,16 @@ if (onnxruntime_BUILD_CSHARP)
 #  set_property(GLOBAL PROPERTY VS_DOTNET_TARGET_FRAMEWORK_VERSION "netstandard2.0")
   include(onnxruntime_csharp.cmake)
 endif()
+
+if (onnxruntime_BUILD_FOR_WINDOWS_STORE)
```

Comment:
can you comment this section? why do store apps need different binaries, why does x86 need SEH, etc.

Reply:
Especially when we already have a WINDOWS_STORE flag, that is used (sometimes set false, sometimes set true) .

How is BUILD_FOR_WINDOWS_STORE different from WINDOWS_STORE ?

---
In reply to: [414718936](https://github.com/microsoft/onnxruntime/pull/3684#discussion_r414718936) [](ancestors = 414718936)

Reply:
Maybe have CMake's support for the store and adding a onnxruntime_APP_CONTAINER flag might be clearer

Reply:
the WINDOWS_STORE setting is supposed to be the whole shebang. Make it work in the store.
The way we use WINDOWS_STORE is kinda funky, because we override its setting of AppContainerApplication and set it to false.

set(CMAKE_VS_GLOBALS "AppContainerApplication=false")

This has the behavior of still building the application with the wcos libs, but not for an appcontainer.

So onnxruntime_BUILD_FOR_WINDOWS_STORE matches the intention of WINDOWS_STORE, which is to make the applications build for the WINDOWS_STORE (appcontainer and all).